### PR TITLE
CONSOLE-3824: Remove kubevirt-plugin code

### PR DESCRIPTION
**Pull request by pcbailey**
This PR does the following:

- Removes the kubevirt-plugin package and references in configuration files
- Copies code upon which the topology plugin is dependent from the kubevirt-plugin package to the topology package
- Changed the i18n prefix in the network-attachment-definition-plugin package from `kubevirt-plugin` to `network-attachment-definition-plugin`

Jira: https://issues.redhat.com/browse/CONSOLE-3824

### CODE REVIEW

**Comment by avivtur** [comment]
This PR will also solve: https://issues.redhat.com//browse/CNV-37607

**Comment by avivtur** [comment]
Excellent work @pcbailey :1st_place_medal: 
/lgtm

**Comment by jhadvig** [comment]
QE Approver:
/assign @yapei 
Docs Approver:
/assign @opayne1 
PX Approver:
/assign @RickJWagner

File: frontend/webpack.config.ts [hunk 1]
```diff
@@ -326,12 +326,14 @@ const config: Configuration = {
...
     new CopyWebpackPlugin([{ from: './packages/gitops-plugin/locales', to: 'locales' }]),
-    new CopyWebpackPlugin([{ from: './packages/kubevirt-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/metal3-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([{ from: './packages/vsphere-plugin/locales', to: 'locales' }]),
     new CopyWebpackPlugin([
       { from: './packages/network-attachment-definition-plugin/locales', to: 'locales' },
     ]),
+    new CopyWebpackPlugin([
+      { from: './packages/network-attachment-definition-plugin/locales', to: 'locales' },
+    ]),
```
Comment (yapei) [inline frontend/webpack.config.ts:336]: are these lines duplicate?

**Comment by yapei** [comment]
1. launched a cluster, subscribed `OpenShift Virtualization` operator successfully
2. Verify `Virtualization` menu exists
3. Verify if secondary menu `Network Attachment Definitions` exist in `Networking` menu
-> No `Network Attachment Definitions` menu, see screen recording, not sure if this is in the scope of this PR

https://github.com/openshift/console/assets/12692381/0c1af56c-3e3b-4af8-808a-d5ed7f6618e9

**Comment by yapei** [comment]
also due to https://issues.redhat.com/browse/OCPBUGS-22853 I can't verify if VM topology codes are working as expected or not(moving away `frontend/packages/kubevirt-plugin/src/topology` to `frontend/packages/topology` )

File: frontend/packages/network-attachment-definition-plugin/locales/en/network-attachment-definition-plugin.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "<0>OpenShift Virtualization Operator</0> or <3>SR-IOV Network Operator </3>needs to be installed on the cluster, in order to pick the Network Type.": "<0>OpenShift Virtualization Operator</0> or <3>SR-IOV Network Operator </3>needs to be installed on the cluster, in order to pick the Network Type.",
+  "Create": "Create",
+  "Cancel": "Cancel",
+  "Resource name": "Resource name",
+  "VLAN tag number": "VLAN tag number",
+  "IP address management": "IP address management",
+  "Bridge name": "Bridge name",
+  "MAC spoof check": "MAC spoof check",
+  "Bridge mapping": "Bridge mapping",
+  "Physical network name. A bridge mapping must be configured on cluster nodes to map between physical network names and Open vSwitch bridges.": "Physical network name. A bridge mapping must be configured on cluster nodes to map between physical network names and Open vSwitch bridges.",
```
Comment (opayne1) [inline frontend/packages/network-attachment-definition-plugin/locales/en/network-attachment-definition-plugin.json:29]:
```suggestion
  "Physical network name. Bridge mapping must be configured on cluster nodes to map between physical network names and Open vSwitch bridges.": "Physical network name. Bridge mapping must be configured on cluster nodes to map between physical network names and Open vSwitch bridges.",
```

**Review by opayne1 — COMMENTED** [review]
One small comment but otherwise looks good

/label docs-approved

File: frontend/webpack.config.ts (hunk 1 above)
Comment (pcbailey) [reply frontend/webpack.config.ts]: Yes, they were. I removed the duplicate. Thanks for catching that!

**Comment by pcbailey** [comment]
> 1. launched a cluster, subscribed `OpenShift Virtualization` operator successfully
> 2. Verify `Virtualization` menu exists
> 3. Verify if secondary menu `Network Attachment Definitions` exist in `Networking` menu
>    -> No `Network Attachment Definitions` menu, see screen recording

I fixed this issue. It was looking for the `KUBEVIRT` flag, so I updated it to look for the `KUBEVIRT_DYNAMIC` flag.

File: frontend/packages/network-attachment-definition-plugin/locales/en/network-attachment-definition-plugin.json (hunk 2 above)
Comment (pcbailey) [reply frontend/packages/network-attachment-definition-plugin/locales/en/network-attachment-definition-plugin.json:29]: Updated! Thank you!

**Comment by pcbailey** [comment]
> also due to https://issues.redhat.com/browse/OCPBUGS-22853 I can't verify if VM topology codes are working as expected or not(these codes moved away from `frontend/packages/kubevirt-plugin/src/topology` to `frontend/packages/topology` ) since `VirtualMachines` page is not accessible

I'm not sure I understand what you mean. That bug pertains to OCP 4.14/CNV 4.13. I also confirmed that the root cause of that bug is no longer present in our code from 4.14 onward. I was able to test the Topology page and Virtual Machines pages. Could you please clarify the issue you mention here? Thanks!

**Comment by yapei** [comment]
I still can't visit VirtualMachines page and other pages(OpenShift Virtualization 4.14.0 installed), here is the recording

https://github.com/openshift/console/assets/12692381/8088a280-81bf-46fd-8be6-22e935d30231

**Comment by pcbailey** [comment]
> I still can't visit VirtualMachines page and other pages(OpenShift Virtualization 4.14.0 installed), here is the recording

Ah, I understand now. This PR should be tested on the latest versions of OCP and OpenShift Virtualization.

**Comment by yapei** [comment]
generally the changes look good from QE's point of view
/label qe-approved

**Review by jhadvig — APPROVED** [review]
/approve

**Pull request merged**